src/seat-local.c

```diff
@@ -158,6 +158,8 @@ get_vt (SeatLocal *seat, DisplayServer *display_server)
             g_signal_connect (display_server, DISPLAY_SERVER_SIGNAL_READY, G_CALLBACK (display_server_ready_cb), seat);
             g_signal_connect (display_server, DISPLAY_SERVER_SIGNAL_STOPPED, G_CALLBACK (display_server_transition_plymouth_cb), seat);
             plymouth_deactivate ();
+
+            return vt;
```

Comment:
Is the problem here that Plymouth is being quit instead of waiting for it to quit? We should check the tests to fix why they aren't detecting this / make them do that. Also I think the else branch below this one should probably also not quite Plymouth.

Can you put this into a separate PR with a better commit description.

Reply:
Although I agree a theoretically cleaner fix would be nice, given the limited maintenance status of the project, wouldn't it be better to have some form of a fix than none?